CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ file is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and
 this project adheres to [Semantic
 Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Changed
+
+- Change struct `Configuration` to enum `Configs` and define configuration by
```

Comment:
Could you change this to past tense please?

Reply:
Sure, done!